specfile/sourcelist.py

```diff
@@ -0,0 +1,116 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import collections
+from typing import List, Optional, overload
+
+from specfile.rpm import Macros
+from specfile.sections import Section
+from specfile.tags import Comments
+
+
+class Source:
+    """
+    Class that represents a spec file source/patch.
+
+    Attributes:
+        url: Literal URL of the source/patch as stored in the spec file.
+        comments: List of comments associated with the source/patch.
+    """
+
+    def __init__(self, url: str, comments: Comments) -> None:
+        self.url = url
```

Comment:
Yes, it can, but URL can point to a local file as well, e.g. `file:///dev/null`. But of course this form isn't actually used in spec files, `url` just seemed a better fit than `path`. Can we come up with a better word for this? `location`?

Reply:
`location` sounds good to me, it's the most general thing and can mean both a file and a URL so I'd personally go for that unless we can think of something completely different. But `location` is in my opinion clearer than `url`